### PR TITLE
New article: Moving large volumes of data to S3

### PR DESCRIPTION
**Pull request by jacobtomlinson**

### CODE REVIEW

File: _posts/2017/2017-04-20-moving-70-terrabytes-to-s3.md [hunk 1]
```diff
@@ -31,35 +31,33 @@ In a word, [sneakernet][xkcd-sneakernet].
...
 
 #### Installing the device
 
-To get the best performance from the device you need to connect it to a 10gbps copper or optical SFP+ port on your network. You need a rather powerful workstation or client to run the software and that needs to be connected to the network with a 10gbps connection. The workstation/client also needs to have access to the data via a dedicated 10gbps link to ensure there are no bottlenecks to slow down the transfer.
+To get the best performance from the device you need to ensure there are no bottlenecks to slow down the transfer. Every connection to connection between the Snowball and the storage much be 10gbps or higher. Therefore it nust connect to a 10gbps copper or optical SFP+ port on your network. You need a rather powerful workstation or client to run the software and that needs to be connected to the network with a 10gbps connection. The workstation/client also needs to have access to the data via a dedicated 10gbps link.
```
Comment (niallrobinson) [inline _posts/2017/2017-04-20-moving-70-terrabytes-to-s3.md]: this is good, but there are some typos: "nust" "every connection to connection"

File: _posts/2017/2017-04-20-moving-70-terrabytes-to-s3.md [hunk 2]
```diff
@@ -31,35 +31,33 @@ In a word, [sneakernet][xkcd-sneakernet].
...
 
 #### Running the transfer
 
-We decided to copy the data using the snowball client, which in hindsight was probably as a mistake as it seems to be unofficially deprecated in favour of the s3 adapter. Sadly we came to this realisation too late and didn't want to complicate things by changing tool mid transfer.
+We decided to copy the data using the Snowball client, which in hindsight was probably as a mistake as it seems to be unofficially deprecated in favour of the s3 adapter. Sadly we came to this realisation too late and didn't want to start over by changing tool mid transfer.
```
Comment (niallrobinson) [inline _posts/2017/2017-04-20-moving-70-terrabytes-to-s3.md]: "start again" - you're not American (kids these days) ;)

File: _posts/2017/2017-04-20-moving-70-terrabytes-to-s3.md [hunk 3]
```diff
@@ -31,35 +31,33 @@ In a word, [sneakernet][xkcd-sneakernet].
 
 It sounds like a joke at first, but when you do the maths it turns out that copying data to a hard drive and transporting it somewhere else is still a pretty efficient way to move large amounts of data around.
 
-Luckily for us Amazon Web Services provide a robust and secure way of doing this called [AWS Snowball][aws-snowball]. Taking it's name from their archive storage system [AWS Glacier][aws-glacier] it is a service for importing big chunks of data into their cloud storage. It works by posting you a very large and durable hard drive appliance, you simply fill the drive up with data and then ship it back to them.
+Luckily for us Amazon Web Services provide a robust and secure way of doing this called [AWS Snowball][aws-snowball]. Taking its name from their archive storage system [AWS Glacier][aws-glacier] Snowball is a service for importing big chunks of data into their cloud storage. It works by posting you a very large and durable hard drive appliance, you simply fill the drive up with data and then ship it back to them.
```
Comment (niallrobinson) [inline _posts/2017/2017-04-20-moving-70-terrabytes-to-s3.md]: this really has to have a comma now: "archive storage system AWS Glacier, Snowball is a"

**Comment by niallrobinson** [comment]
looking good after those last couple of tweaks

**Comment by jacobtomlinson** [comment]
@niallrobinson Done! Please merge.

**Pull request merged**